src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    private static final Color[] colors = Color.values();
```

Comment:
this array can be a local variable inside a method. 
Usually, class-level variables are some services or fields that are part of a bigger entity, like engine is a part of a car, or an email is a detail of some user, but an array like this is more of a part of a method and is not worth to be stored in a class, even if it is a constant value like in this case

Reply:
not fixed

Reply:
not fixed